tests/src/core/AutomatableModelTest.cpp

```diff
@@ -41,6 +42,13 @@ private slots: // helper slots
 private slots: // tests
 	//! Test that upcast and exact casts work,
 	//! but no downcast or any other casts
+
+	void initTestCase()
+	{
+		using namespace lmms;
```

Comment:
`using namespace` just to write one single line shorter? :smiley:

Reply:
Yeah, if the test case were any bigger, it can get annoying having to type out `lmms::` all the time, and I believe the tests already do this elsewhere.